src/main/java/frc/robot/subsystems/intake/IntakeIOSim.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix6.controls.PositionVoltage;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Velocity;
+import lib.motors.SparkMaxSim;
+import lib.motors.TalonFXSim;
+
+public class IntakeIOSim implements IntakeIO {
+    TalonFXSim angleMotor;
+    SparkMaxSim frontRoller;
+    SparkMaxSim centerRoller;
+    PositionVoltage positionRequest = new PositionVoltage(0);
+    PIDController angleController =
```

Comment:
The controller inside the TalonFXSim class, no need to make it a member.

Reply:
OK

Reply:
87e7e5cac0b80010fdf2f71007ae5eb93d563786